Extensions/PanelSpriteObject/panelspriteruntimeobject.js

```diff
@@ -3,39 +3,72 @@
  *  2013 Florian Rival (Florian.Rival@gmail.com)
  */
 
+//TODO Someone check tiled and texture typing, I am not sure about it.
+/**
+ * @typedef {Object} panelSpriteObjectDataType
+ * @property {number} rightMargin The right margin
+ * @property {number} leftMargin The left margin
+ * @property {number} topMargin The top margin
+ * @property {number} bottomMargin The bottom margin
+ * @property {PIXI.Sprite} [tiled] The tiled PIXI sprite
+ * @property {number} width The objects width
+ * @property {number} height The objects height
+ * @property {PIXI.BaseTexture} texture The objects texture
```

Comment:
`string` (should have been called `textureName` ideally)

Reply:
fixed